content/kotlin/concepts/hashmap/terms/put/put.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.put()'
+Description: 'Used to add a key-value pair to a map.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'HashMap'
+  - 'Data Types'
+  - 'Collections'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+In Kotlin, the `put()` function is used to add a key-value pair to a map. It is a member function of the `MutableMap` interface, which means it can be called on any object that implements this interface, such as a `HashMap` or `LinkedHashMap`.
+
+## Syntax
+
+```pseudo
+fun <K, V> MutableMap<K, V>.put(key: K, value: V): V?
+```
+
+It takes in two parameters:
+
+- key: the key of the key-value pair to add or update in the map.
+- value: the value of the key-value pair to add or update in the map.
```

Comment:
```suggestion
- The `key` of the key-value pair to add or update in the map.
- The `value` of the key-value pair to add or update in the map.
```

We can also add on here what the functionality is for the key and value. For example, the key is a unique identifier that is used to access certain values within the hash map and the value is what's actually being stored or the data that associated with the key.